### DISCUSSION TITLE
Documentation update to bring clarity

### DISCUSSION THREAD

**Discussion by mikaelsand** [qetza/replacetokens-task#56] [Ideas]
When I started using this repo in my Azure Pipelines I could not find anything in the documentation about _where_ the task gets its values from. There is a lot of info on how you point to files that should have tokes replaced, but there is no mention of variable groups anywhere. I found the information in another article.

Please add the information on where the task gets its values.

**Comment by qetza** [comment]
Hi @mikaelsand,
Thanks for the feedback, I'll update the documentation to make it clearer that the variables are from the inline variables and variable groups declare in the pipeline definition.
